submissions/nadsatt/memory-pair-game/index.js

```diff
@@ -0,0 +1,285 @@
+const doc = document.documentElement,
```

Comment:
It's not a big blocker, but better to use appropriate const names.
JS has convention regarding global constants - NAMES_SHOULD_LOOK_LIKE_THIS

Please see examples https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const